ai-platform-unified/notebooks/unofficial/hyperparameter_tuning/distributed-hyperparameter-tuning.ipynb

```diff
@@ -0,0 +1,879 @@
+{
```

Comment:
<p>Puts -&gt; Sets</p><p> </p><p>It is a best practice to start with -&gt; The recommended practice is to start with</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/391/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>